### PR TITLE
add(theme): Teradata material theme

### PR DESCRIPTION
**Pull request by kyleledbetter**
## Test (locally)
- [ ] Build sphinx docs

Should look like:

teradata-presto-theme

### CODE REVIEW

**Comment by cawallin** [comment]
Is there any reason for "view page source"? It doesn't seem necessary.

File: presto-docs/src/main/sphinx/themes/sphinx_rtd_theme/versions.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <dt>On Read the Docs</dt>
+          <dd>
+            <a href="//{{ PRODUCTION_DOMAIN }}/projects/{{ slug }}/?fromdocs={{ slug }}">Project Home</a>
+          </dd>
+          <dd>
+            <a href="//{{ PRODUCTION_DOMAIN }}/builds/{{ slug }}/?fromdocs={{ slug }}">Builds</a>
+          </dd>
+      </dl>
+      <hr/>
+      Free document hosting provided by <a href="http://www.readthedocs.org">Read the Docs</a>.
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/themes/sphinx_rtd_theme/versions.html]: This isn't hosted on readthedocs.org, though?

**Comment by cawallin** [comment]
Also, can you add a scroll bar for the index pane? You can't easily scroll there.

Otherwise, though, this looks awesome!

**Comment by cawallin** [comment]
One more thing -- it could be that the test version that @mattsfuller put up on http://teradata.github.io/presto/docs/staging/index.html isn't set up right, but search doesn't work.

**Comment by mattsfuller** [comment]
Looks awesome. my only comment was about the search not working. not sure if I did something wrong.

**Comment by kyleledbetter** [comment]
@mattsfuller @cawallin the search works locally for me but is throwing a jquery error on this function on the hosted site:
`.on('click', "[data-toggle='wy-nav-top']", function() {`

I grabbed an existing sphinx theme so would need to dig into debug, but I'm guessing like most jquery errors it's an order of javascript files issue (i've moved on to angularjs years ago so need to relearn jquery hijinks :) )

File: presto-docs/src/main/sphinx/themes/sphinx_rtd_theme/versions.html (hunk 1 above)
Comment (kyleledbetter) [inline presto-docs/src/main/sphinx/themes/sphinx_rtd_theme/versions.html]: nah, this is a fork of an existing open source sphinx theme (https://github.com/snide/sphinx_rtd_theme) we can remove that

**Comment by kyleledbetter** [comment]
@cawallin not sure what you meant by `scroll bar for the index pane`, the left sidenav and right content area scroll fine for me. Maybe a screenshot might help? thx

**Comment by cawallin** [comment]
It looks the same for me as in your gif. They both scroll ok, but for the left sidenav, I have to scroll with the wheel of my mouse; there isn't any scroll bar that I could click with my mouse if I wanted.
websitescreenshot

**Comment by kyleledbetter** [comment]
@cawallin hmmm, i never see scrollbars on osx anyways... when i hover on the left nav i'm able to scroll, are you not able to on hover?

**Comment by cawallin** [comment]
Hmmm no, still have to use the mouse wheel, but I'm on Linux (both Firefox and Chrome), so it might not be the most representative. You can still navigate via the breadcrums, so it's probably fine.

**Comment by kyleledbetter** [comment]
So here's the theme i'm using, i'm assuming the results will be the same for you http://docs.readthedocs.org/en/latest/

All I did was small color css update, but I can dig in and actually alter the layout CSS to make it more accessible

**Comment by cawallin** [comment]
If that's how readthedocs.org formats their docs, then it's probably fine :) I'm not a UX designer by any means. Thanks for looking into it!

**Comment by mattsfuller** [comment]
@cawallin @arhimondr can you review?
@BrendaNoonan Preview of docs are staged here: http://teradata.github.io/presto/docs/preview/

**Comment by mattsfuller** [comment]
@kyleledbetter could you put the Presto logo on the header somewhere?
It looks like this: https://avatars1.githubusercontent.com/u/6882181?v=3&s=400
You can see it on the main docs: https://prestodb.io/

**Comment by cawallin** [comment]
Looks good!

**Comment by mattsfuller** [comment]
ok. I'll close this PR and merge onto our release.

**Pull request closed without merging**